Description: De-optify the path for xapi create_templates script.
Forwarded: not-needed
Author: Citrix.
--- a/61-regenerate-old-templates
+++ b/61-regenerate-old-templates
@@ -64,7 +64,7 @@
 regenerate_templates(){
     echo "Regenerating built-in templates"
     logger -t ${TAG} "Regenerating built-in templates"
-	/opt/xensource/libexec/create_templates
+	/usr/lib/xen-common/xapi/libexec/create_templates
 }
 
 start() {
